### ISSUE TITLE
Line containing both error and warning is prefixed with orange dot 

### ISSUE THREAD

**Issue by riophae** [hebaishi/linter-gcc#79]
## Checklist
* [x] Check that you have the GNU C Compiler installed by typing ```gcc --version``` into a terminal.
* [x] If you don't have the ```linter``` package installed, either update to the latest version of ```linter-gcc``` or install linter directly with ```apm install linter```.
* [x] Ensure that you are using a valid grammar - either C/C++/C++14
* [x] Ensure that you have read all the relevant information in the [wiki](https://github.com/hebaishi/linter-gcc/wiki), particularly the [Common Issues](https://github.com/hebaishi/linter-gcc/wiki/Common-Issues) page.

## Issue (including debugging messages):
Enter these lines of code:

```cpp
int main() {
  int *const p;
  return 0;
}
```

Then `linter-gcc` catches one error and one warning:

```
[GCC][error]uninitialized const `p`[-fpermissive] at line 3 col 14
[GCC][warning]unused variable `p`[-Wunused-variable] at line 3 col 14
```

So far everything is all right. 

The problem is that, the line 3 where error and warning occur is prefixed with an ORANGE dot, indicating only there is warning. So the user might not realize that there is also error and would not pay enough attention to.

![screenshot](https://cloud.githubusercontent.com/assets/1289884/18411978/b1a43d10-77b6-11e6-828d-4c09fe3758d6.png)

It's not a serious issue, but worth consideration in my opinion.

**Comment by Arcanemagus** [comment]
This is a bug in [Linter](https://github.com/steelbrain/linter), which is already fixed in the upcoming UI rewrite ([linter-ui-default](https://github.com/steelbrain/linter-ui-default)).

**Comment by riophae** [comment]
Got it. Thank you!
